### DISCUSSION TITLE
Deploying Micronaut application to vercel

### DISCUSSION THREAD

**Discussion by Kapsule98** [vercel/community#1097] [Help] [answered]
I am planning to create a micronaut project for learning purposes.
I wanted to know if its possible to deploy micronaut application to vercel.

If its possible is there a guide or deployment steps 

thanks in advance

**Comment by swarnava** [comment, marked as answer]
Micronaut is a JVM based framework. Vercel doesn't support Java: https://vercel.com/docs/concepts/functions/serverless-functions/runtimes
